deploy/backplane/acs-integration/config.yaml

```
@@ -0,0 +1,5 @@
+deploymentMode: "SelectorSyncSet"
+selectorSyncSet:
+  matchLabels:
+    api.openshift.com/addon-acs-fleetshard-dev: "true"
```

```suggestion
    api.openshift.com/addon-acs-fleetshard-dev: "true"
    api.openshift.com/addon-acs-fleetshard-qe: "true"
```

Or don't need qe env for this?